check-google-java-format.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
```

Comment:
Given the compatibility changes, I think that none of these changes to the shebang line are necessary any more.  Is that correct?